### DISCUSSION TITLE
Add ability to import json files as variables in editor.

### DISCUSSION THREAD

**Discussion by Sapper-Morton** [4ian/GDevelop#6858] [Enhancement ideas (and their technical discussions)]
I'd like to see in-editor support to import JSON structures as variables so I wouldn't need to load them one by one while running the game.
My current project requires a lot of JSON loading and once they're locked in I'd rather have them permanently in my global variables than in files inside the game directory.

Meanwhile the feature is not available I'll be hand editing the game.json file to add the structures to the global variables.

**Comment by 4ian** [comment]
It sounds sensible to have a import/export to JSON button in variables editor. One point where we would need to be careful is to ensure "refactoring" is done across the project if a variable type is changed.

**Reply by Silver-Streak** [reply]
With an import/export option, and a little bit of prepping, game devs could have an easy way to export/import data to CastleDb and manage it in bulk for things like large item lists, which is exceptionally difficult to do in the current variable editor.

While I'd love some tweaks to the vatiable editor for db style needs, this is the next best thing.
